pkg/client/cmd/config.go

```diff
@@ -159,6 +172,35 @@ func setCluster(cmd *cobra.Command, args []string) {
 	}
 }
 
+// rmCluster removes a server from the config file
+func rmCluster(cmd *cobra.Command, args []string) {
+	if len(args) == 0 || args[0] == "" {
+		cmd.Usage()
+		return
+	}
+
+	name := args[0]
+	if name == "" {
```

Comment:
Hm, I think you don't need to check if `args[0]` is empty again, you already did that on line 177.